rts/lra/lra-proxy/api/src/main/java/io/narayana/lra/client/internal/proxy/nonjaxrs/LRACDIExtension.java

```diff
@@ -101,12 +102,12 @@ public void observe(@Observes AfterBeanDiscovery event, BeanManager beanManager)
      * @return Collected methods wrapped in {@link LRAParticipant} class or null if no non-JAX-RS methods have been found
      */
     private LRAParticipant getAsParticipant(ClassInfo classInfo) throws ClassNotFoundException {
-        if (!isLRAParticipant(classInfo)) {
+        Class<?> javaClass = getClass().getClassLoader().loadClass(classInfo.name().toString());
```

Comment:
There is a risk that `Class.getClassLoader()` will be null if the class was loaded by bootstrap class loader. Are we sure that this never happens? I understand that this code snippet was already involved in the code base - this PR does not add it. But just wondering.

Reply:
@ochaloup not sure TBH. Maybe worth exploring in the new JBTM?

Reply:
+1

Reply:
The bootstrap class loader loads system classes but the CDI extension is not a system class so a null check is not necessary.

Even if this were not the case, then why create a JIRA rather than updating the PR by checking for null, the code already handles null values on the next line (but this is really not necessary).